src/components/Character/Character.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './Character.css';
+
+export default function Display({ shirt, pant, shoe }) {
+  return (
+    <div className="display-fit">
+      <div>
+        <img src={`${process.env.Public_URL}/items/Shirts/${shirt}-shirt.png`} />
```

Comment:
I believe `PUBLIC_URL` is the name of the variable. You have it here as `Public_URL` which is undefined. Fixing this in your image src should get the images to load correctly.